e2e/testcases/v2-advanced-search/2-search-birth-event-declaration-child-details.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { expect, test, type Page } from '@playwright/test'
+import { getToken, loginToV2 } from '../../helpers'
+import { createDeclaration } from '../v2-test-data/birth-declaration'
+import { CREDENTIALS } from '../../constants'
+import { faker } from '@faker-js/faker'
+
+/**
+ * Converts a numeric month value (1–12) to its corresponding short English month name (e.g., "Jan", "Feb").
+ *
+ * @param {number} month - The month number (1 for January, 12 for December).
+ * @returns {string} The short name of the month in English.
+ *
+ * @example
+ * getMonthShortName(1); // returns "Jan"
+ * getMonthShortName(12); // returns "Dec"
+ */
+const getShortMonthName = (month: number) => {
+  const arbitraryDate = new Date(2000, month - 1)
+  return arbitraryDate.toLocaleString('en-US', { month: 'short' })
+}
+
+test.describe
+  .serial('Advanced Search - Birth Event Declaration - Child details', () => {
+  let page: Page
+  let [yyyy, mm, dd] = ['', '', '']
+  let fullNameOfChild = ''
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage()
+    const token = await getToken(
+      CREDENTIALS.LOCAL_REGISTRAR.USERNAME,
+      CREDENTIALS.LOCAL_REGISTRAR.PASSWORD
+    )
+
+    const record: Awaited<ReturnType<typeof createDeclaration>> =
+      await createDeclaration(token, {
+        'child.dob': faker.date
+          // Randomly chosen DOB between 2010-01-01 and 2020-12-31
+          // Ensures the created record appears on the first page of search results
+          .between({ from: '2010-01-01', to: '2020-12-31' })
+          .toISOString()
+          .split('T')[0],
+        //@ts-ignore
+        'child.reason': 'Other', // needed for late dob value
+        'child.gender': 'female'
+      })
+    ;[yyyy, mm, dd] = record.declaration['child.dob'].split('-')
+    fullNameOfChild =
```

Comment:
you can use template literal, e.g. `${record.declaration['child.firstname]} ${record.declaration['child.surname']}` 

there also exists a `formatName()` helper which probably could be used here